### PR TITLE
[4.0] Namespace libraries\cms\html

### PR DESCRIPTION
**Pull request by brianteeman**
Continuing the work to namespace everything

### CODE REVIEW

File: libraries/cms/html/batch.php [hunk 1]
```diff
@@ -25,13 +27,13 @@
 	 *
 	 * @since       1.7
 	 *
-	 * @deprecated  4.0 instead of JHtml::_('batch.access'); use JLayoutHelper::render('joomla.html.batch.access', array());
+	 * @deprecated  4.0 instead of JHtml::_('batch.access'); use LayoutHelper::render('joomla.html.batch.access', array());
 	 */
 	public static function access()
 	{
 		JLog::add('The use of JHtml::_("batch.access") is deprecated use JLayout instead.', JLog::WARNING, 'deprecated');
```
Comment (Quy) [inline libraries/cms/html/batch.php]: Update with `use Joomla\CMS\Log\Log;`
Change `JLayout `

Comment (Quy) [reply libraries/cms/html/batch.php]: Change to LayoutHelper?

Comment (brianteeman) [reply libraries/cms/html/batch.php]: already resolved isnt it?

Comment (Quy) [reply libraries/cms/html/batch.php]: This part: use JLayout instead

Comment (brianteeman) [reply libraries/cms/html/batch.php]: Ah you mean in the Log message - I deliberately left that

**Comment by brianteeman** [comment]
(currently unable to connect to ci.joomla.org to see what the drone errors are)

**Comment by brianteeman** [comment]
Drone error is unrelated to this pr

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**